README.md

```diff
@@ -229,6 +229,27 @@ Even if you don't have any errors in your IDE, `json2typescript` will not proper
 
 # Detailed reference
 
+## Property declarations
+
+For class properties to be visible to the mapper they **must be initialized**, otherwise they are ignored. They can be initialized using any (valid) value or `undefined`.
+
+```typescript
+@JsonObject("User")
+export class User {
+    @JsonProperty("name", String, false)  // ✔ Decorator
+    name: string = "";  ✔ Initialization
+    
+    @JsonProperty("alias", string, false)  // ❌ Must use String instead.
+    alias: string = ""; //  ✔ Initialization
+    
+    @JsonProperty("expertise", String, false) ✔ Decorator
+    expertise: string;  //  ❌ Must be initialised, otherwise is ignored.
+}
+```
+
+> **Warning**: Non initialized properties won't trigger any exception, as **they are invisible to the mapper**.
+
+
```

Comment:
Can you please move your text to the property decorators section? It makes more sense there like this (the italic part is your text):

Property decorators

Property decorators are a vital part for type checking. It is important that the type in the decorator matches the TypeScript type.

_For class properties to be visible to the mapper they **must be initialized**, otherwise they are ignored. They can be initialized using any (valid) value or `undefined`._

```typescript
@JsonObject("User")
export class User {
    @JsonProperty("name", String, false)  // ✔ Decorator
    name: string = ""; // ✔ Initialization
    
    @JsonProperty("alias", string, false)  // ❌ Must use String instead.
    alias: string = ""; // ✔ Initialization
    
    @JsonProperty("expertise", String, false) // ✔ Decorator
    expertise: string;  // ❌ Must be initialised, otherwise is ignored.
}
```

Remark: Replaced the original code because it is not necessary anymore. Added missing `//` after the name declaration and the last decorator.